apps/src/applab/dataBrowser/dataStyles.js

```diff
@@ -76,5 +100,9 @@ export {
   button,
   editButton,
   rightButton,
-  input
+  input,
+  viewHeader,
+  backLink,
+  debugLink,
+  debugData,
```

Comment:
should we just export each item individually instead of exporting an object full of these at the end? i.e.
```
export const debugData;
```

Then the consumer can either do:
```
import { debugData } from './dataStyles];
```
or
```
import * as dataStyles from './dataStyles}
```